Project-Aurora/Project-Aurora/Settings/VariableRegistry.cs

```diff
@@ -231,13 +288,12 @@ public void RemoveVariable(string name)
 
         public object Clone()
         {
-            string str = JsonConvert.SerializeObject(this, Formatting.None, new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.All, Binder = Aurora.Utils.JSONUtils.SerializationBinder });
-
+            string str = JsonConvert.SerializeObject(this, Formatting.None, new JsonSerializerSettings {TypeNameHandling = TypeNameHandling.All, Binder = Aurora.Utils.JSONUtils.SerializationBinder});
             return JsonConvert.DeserializeObject(
-                    str,
-                    this.GetType(),
-                    new JsonSerializerSettings { ObjectCreationHandling = ObjectCreationHandling.Replace, TypeNameHandling = TypeNameHandling.All, Binder = Aurora.Utils.JSONUtils.SerializationBinder }
-                    );
+                str,
+                this.GetType(),
+                new JsonSerializerSettings {ObjectCreationHandling = ObjectCreationHandling.Replace, TypeNameHandling = TypeNameHandling.All, Binder = Aurora.Utils.JSONUtils.SerializationBinder}
+            );
```

Comment:
You really need to stop committing code that changes styling changes like this. It's unnecessary, and the project's general style is to have spaces around the braces. Even if your IDE is doing these changes automatically, you should be looking at what you are committing before doing so intead of just adding all changes.

Reply:
Honestly you guys should have looked at supplying a ReSharper settings file that i could use so i don't have these issues in the repo.

Reply:
I don't think it's really on us to have to provide a file for a 3rd party proprietry software.

Reply:
ReSharper isn't a proprietry software and is actual being used quite the bit in actual companies. There exists a 30 day trail for it meaning it can be configured up to how the code should be removing these issues later down the line.

Reply:
Right, but if you wanna keep using it beyond the 30 days you need to pay for it. Therefore, proprietry.

Either way, if it is or isn't on us to provide a config file, it doesn't mean it's open to random style changes that add nothing.